app/controllers/kaui/accounts_controller.rb

```diff
@@ -107,6 +111,8 @@ def show
     wait(fetch_payment_methods)
     @payment_methods = wait(fetch_payment_methods_with_details).map { |pm_f| pm_f.execute }.map { |pm_f| wait(pm_f) }.reject { |pm| pm.nil? }
     @available_tags = wait(fetch_available_tags)
+    @children = wait(fetch_children)
```

Comment:
I am not super familiar with promises. Question: When will the call to fetch all the children accounts be executed? Is that when the user clicks on the `Children(N)` link?

Reply:
The promise here is only to check if the account has children, . When the user clicks the `Children(N)` link appear the list screen, and then via DataTable Ajax call it's fetches the children, but it is not done by the promise.